### PR TITLE
Run RPi Python script from Node process

### PR DESCRIPTION
**Pull request by ssg-luke**
Needs testing on the RPi before merging

### CODE REVIEW

**Comment by lukefanning** [comment]
We might want to think about removal of node-ipc and just use python-shell's stdin & stdout streams as well. Speed should be about the same I think.

File: rio-server/send-data.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ const dgram = require('dgram');
 const config = require('./config');
 const uuid = require('node-uuid');
 const consoleOutput = require('./outputs/console-output');
-const piOutput = require('./outputs/pi-output');
+if (config.sendToPi) {
```
Comment (kyle-ssg) [inline rio-server/send-data.js]: Does this actually do anything?

Comment (lukefanning) [inline rio-server/send-data.js]: Yeah without it the server always tried to start the python script which crashes the node process unless you're on a RPi

Comment (kyle-ssg) [inline rio-server/send-data.js]: Gotcha

**Pull request merged**